### PR TITLE
Add EIP: NFT Creator Attribution

### PR DESCRIPTION
**Pull request by strollinghome**
Link to discussion: https://ethereum-magicians.org/t/eip-authorship-attribution-for-erc721/14244

### CODE REVIEW

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: EIPS/eip-7015.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+);
+```
+
+Note that although a `chainId` is necessary for ERC-721 signatures, we omit the parameter from the event as it can be inferred through the transaction data. The `token` parameter MUST match the address of the `emitter` field of the event.
+
+A platform can verify the validity of authorship attribution by reconstructing the signature digest with the parameters emitted and recovering the signer from the `signature` parameter. The `author` parameter MUST match the recovered signer from the `signature` parameter. After verification, the platform can safely attribute authorship to the `author` instead of the account that submitted the transaction.
+
+### Reference Implementation.
+
+See [authorship-attribution](https://github.com/strollinghome/authorship-attribution).
```
[the remark this answers is not in the document]
Comment (strollinghome) [reply EIPS/eip-7015.md]: Updated this, curious what else I need to do to get this merged?

File: EIPS/eip-7015.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+Note that although the `chainId` parameters is necessary for [EIP-712](./eip-712.md) signatures, we omit the parameter from the event as it can be inferred through the transaction data. Similarly, the `token` parameter in the `CreatorAttribution` stuct MUST match the address of the `emitter` field of the event, hence we omit it as well.
+
+A platform can verify the validity of the creator attribution by reconstructing the signature digest with the parameters emitted and recovering the signer from the `signature` parameter. The `creator` parameter MUST match the recovered signer from the `signature` parameter. After verification, platforms can safely attribute the `creator` address as the creator instead of the account that submitted the transaction.
+
+### Reference Implementation
+
+#### Example signature validator
+
+```solidity
+// SPDX-License-Identifier: MIT
```
Comment (SamWilsn) [inline EIPS/eip-7015.md:80]: Code inline in the EIP must be licensed under `CC0-1.0`. Code in the assets directory can be `MIT` if you'd like.

Comment (strollinghome) [reply EIPS/eip-7015.md:80]: Fixed!

File: EIPS/eip-7015.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+---
+eip: 7015
+title: NFT Creator Attribution
+description: An extension of ERC-721 for creator attribution.
```
[the remark this answers is not in the document]
Comment (strollinghome) [reply EIPS/eip-7015.md:4]: Hah true, thanks!

**Review by Mikh333 — APPROVED** [review]

**Pull request merged**